app/templates/road-sign-concepts/road-sign-concept/related.hbs

```diff
@@ -0,0 +1,158 @@
+{{page-title (t "utility.related-traffic-signs")}}
+<BreadcrumbsItem as |linkClass|>
+  <LinkTo
+    @route="road-sign-concepts.road-sign-concept.related"
+    class={{linkClass}}
+  >
+    {{t "utility.related-traffic-signs"}}
+  </LinkTo>
+</BreadcrumbsItem>
+
+<AuBodyContainer @scroll={{true}} class="au-c-body-container--tables">
```

Comment:
The scroll seems broken here, although you added `@scroll={{true}}`. I think the issue is in the SidebarOverlayContainer component. Try adding 
```css
.sidebar-overlay-container {
  overflow-y: scroll;
}
```